streamwebs_frontend/streamwebs/templates/streamwebs/datasheets/canopy_cover_edit.html

```diff
@@ -7,56 +7,53 @@
 
 {% block content %}
 <style>
-    #canopies {
-        cursor: default;
```

Comment:
Personally, I'd prefer to keep this, because it gives the CC squares the Look & Feel of a button rather than a text box.